docs/pages/reference/access-controls/access-lists.mdx

```diff
@@ -190,3 +190,23 @@ above) and run `tctl create <filename>`. Access Lists can be updated by using `t
 Through these you can list Access Lists, get information about a particular Access Lists, and manage
 Access List users. To see more details, run `tctl acl --help`. More detail can be seen in the
 [CLI Reference](../cli/cli.mdx).
+
+## Finding parent Access Lists
+
+In some environments, Access Lists can be nested, where one list owns or references another. 
+When multiple layers of Access Lists are involved, identifying all parent lists can become time-consuming.
```

Comment:
We wouldn't need this paragraph if this section were under "Nested Access Lists". I also don't think we need to say that finding parent lists can be time consuming, since we are giving the reader a script to use to find them quickly.